packages/notebook-extension/src/index.ts

```diff
@@ -2137,6 +2142,29 @@ function addCommands(
     },
     isEnabled
   });
+  commands.addCommand(CommandIDs.renderSideBySide, {
+    label: trans.__('Render Side-by-side'),
+    execute: args => {
+      const current = getCurrent(tracker, shell, args);
+      if (current) {
+        if (!(app instanceof JupyterLab)) {
+          return NotebookActions.renderSideBySide(current);
+        }
+        return NotebookActions.renderSideBySide(current, app);
+      }
+    },
+    isEnabled
+  });
+  commands.addCommand(CommandIDs.renderNotSideBySide, {
+    label: trans.__('Render Not Side-by-side'),
```

Comment:
A better phrasing than 'Render Not Side-by-side' ?

Reply:
Maybe "Render in columns vs "Rend in row"

Reply:
Maybe make it a single column with a toggle instead?

Reply:
Agree, single command with a toggle seems intuitive to me as well. Will start to work on that